### PR TITLE
LPS-96481

### PR DESCRIPTION
**Pull request by hhuijser**

### CODE REVIEW

File: commerce-inventory-api/bnd.bnd [hunk 1]
```diff
@@ -2,7 +2,7 @@ Bundle-Name: Liferay Commerce Inventory API
 Bundle-SymbolicName: com.liferay.commerce.inventory.api
 Bundle-Version: 1.0.0
 Export-Package:\
-com.liferay.commerce.inventory.constants,\
+com.liferay.commerce.inventory.constants
```
Comment (ethanbustad) [inline commerce-inventory-api/bnd.bnd:5]: @hhuijser is this right?

File: commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPFriendlyURLEntryLocalServiceImpl.java [hunk 2]
```diff
@@ -322,13 +322,8 @@ protected void validate(
...
 		if (classPK > 0) {
-			CPFriendlyURLEntry cpFriendlyURLEntry =
-				cpFriendlyURLEntryPersistence.fetchByG_C_C_L_U(
-					groupId, classNameId, classPK, languageId, urlTitle);
-
-			if (cpFriendlyURLEntry != null) {
-				return;
-			}
+			cpFriendlyURLEntryPersistence.fetchByG_C_C_L_U(
+				groupId, classNameId, classPK, languageId, urlTitle);
```
Comment (ethanbustad) [inline commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPFriendlyURLEntryLocalServiceImpl.java:326]: @marco-leo what are we doing here? This looks like a useless fetch -- the new version is functionally the same as the old one, just more obviously useless.

File: commerce-inventory-api/bnd.bnd (hunk 1 above)
Comment (hhuijser) [reply commerce-inventory-api/bnd.bnd:5]: @ethanbustad 
Nope, that's not right. Good catch!
Will look into that one.
Shall I resend in a few separate pulls?

**Comment by hhuijser** [comment]
Will resend

**Comment by hhuijser** [comment]
See
https://github.com/liferay/com-liferay-commerce/pull/1013
https://github.com/liferay/com-liferay-commerce/pull/1014

**Pull request closed without merging**